tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data util-linux_cli-helpers)"
+
+  mkdir -p "${rootfs}/dev"
+  mount --rbind /dev "${rootfs}/dev"
+  echo "ServerName localhost" >> "${rootfs}/etc/apache2/apache2.conf"
+  test "$(chroot "${rootfs}" a2query -M)" = "event"
+  test "$(chroot "${rootfs}" a2dismod mpm_event | head -n 1)" = "Module mpm_event disabled."
+  test "$(chroot "${rootfs}" a2enmod mpm_prefork | grep -o "Enabling module mpm_prefork." )" = \
```

Comment:
I think we should put a comment here explaining why we are disabling the event MPM and enabling the prefork MPM.